### PR TITLE
Update manage-teams-in-modern-portal.md

### PR DESCRIPTION
**Pull request by svpsiva**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-teams-in-modern-portal.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ The grid displays the following properties:
 
 To add a new team, click **Add**. In the **Add a new team** pane, give the team a name and description, set whether you want to make it a private or public team, and set the classification.
 
+> [!NOTE]
+> Newly created teams can be managed right away in the Teams Admin Center, unlike the experience in other clients like, Outlook.
```
Comment (JohanFreelancer9) [inline Teams/manage-teams-in-modern-portal.md:67]:
```suggestion
> Newly created teams can be managed right away in the Teams Admin Center, unlike the experience in other clients, like Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**